### ISSUE TITLE
Tracking Ticket for Requesting New Scrapers

### ISSUE THREAD

**Issue by vpfruitly** [xbapps/xbvr#142]
I thought it might be handy to have a single issue to track requsted scrapers in one place. Here's all the ones requested so far from both here and https://feedback.xbvr.app/

- VR3000
- VirtualRealTrans
- VRBTrans
- VRConk
- SinsVR
- StockingsVR
- StripzVR
- GroobyVR
- VRP Films
- TSVirtualLovers

**Comment by AlmopsVR** [comment]
Can you add https://sexlikereal.com ?

**Comment by vpfruitly** [comment]
I don't think SLR is a studio that produces scenes, they're just a distributor. 

https://www.sexlikereal.com/virtualreality/studios

**Comment by AlmopsVR** [comment]
True, but videos are hosted on the site and there's a lot of content.

**Comment by vpfruitly** [comment]
SLR doesn't host all of the videos from each studio, they only get a selection, so the real solution is to scrape the studio directly. Look at individual videos on SLR and if their studio isn't already a scraper, or listed above, let us know. We can add that studio to the to-do list.

**Comment by Sinoh726** [comment]
hello, can you please add https://wankitnowvr.com 
thank you!

**Comment by dan9qt** [comment]
Might as well add zexyvr.com to go with wankitnowvr.com as they are from the same guys

**Comment by SpedNSFW** [comment]
Two Naughty America companies:
 - https://www.realteensvr.com
 - https://www.randysroadstop.com

Others:
 - https://www.virtualxporn.com
 - https://www.vrclubz.com
 - https://www.littlecaprice-dreams.com/virtual-reality-little-caprice/#

**Comment by Transcodes** [comment]
tonightsgirlfriend.com
vrfirsttimer.com

**Comment by Aerowen** [comment]
@SpedNSFW I've added EvilEyeVR and VirtualXPorn to the SLR scraper. Even though VirtualXPorn does have its own site I found the scene details there are lacking a lot and they're added quickly on SLR so I'm sure that's fine for now.

LittleCapriceVR stopped releasing on SLR after three scenes that were mostly broken and needed fixing by SLR team, I'm not sure if the new ones on her own site are any better? (I don't feel like "renting" her scenes (lol), but that one may need a separate new scraper.

And VRClubz is one of a list of studios that are listed on SLR but aren't actively producing new scenes any more. I've got a whole list of them like; CumCoders, JustVR, VirtualPornDesire, PVRStudio... I'm not sure if those should be added as active scrapers when they're never going to get any new updates. Perhaps there should be a 'scrape only once' setting for scrapers like these studios that stopped producing.

I've added the other ones mentioned to my todo list... most need their own new scraper.

**Comment by trait27** [comment]
No scrapers/matches so far:

"SLR_" indicates that it comes straight from sexlikereal.

Maybe we can activate SLR as a backup scraper (even for "old/paused" studios)...because there are so many scenes from small studios and the catalog is very good (Images, description etc.) and large (if not the best...).

- SLR_BaberoticaVR
- SLR_LustReality
- SLR_BravoModelsMedia
- SLR_TadPoleXXXStudio
- SLR_HoliVR
- SLR_StockingsVR
- SLR_AmateurVR3D
- SLR_AD4X
- SLR_Only3xVR
- SLR_VRedging
- SLR_BurningAngelVR
- SLR_ScrewBoxVR
- SLR_MMM100
- SLR_PVRStudio
- SLR_HerFirstVR
- MatureReality (realitylovers now?)

**Comment by cktger** [comment]
https://jvrporn.com/

**Comment by Deflated-Criossant** [comment]
New site 
https://darkroomvr.com

some scenes are released under virtualtaboo

eg: https://darkroomvr.com/video/i-prescribe-taking-panties-off

filenames : 
drvr-i-prescribe-taking-panties-off-960p.mp4
drvr-i-prescribe-taking-panties-off-1440p.mp4
drvr-i-prescribe-taking-panties-off-psvr_1440p.mp4
drvr-i-prescribe-taking-panties-off-4k.mp4
drvr-i-prescribe-taking-panties-off-5k.mp4
drvr-i-prescribe-taking-panties-off-5k10.mp4
drvr-i-prescribe-taking-panties-off-6k.mp4
drvr-i-prescribe-taking-panties-off-7k.mp4

**Comment by jcgrey2** [comment]
No scrapers yet for the following, and each one has a good number of scenes:

PropertySexVR
PS-Porn
ReeVR
Squeeze VR
VirtualPornDesire
VixenVR

**Comment by vorrac-stash** [comment]
Hi, I just noticed that there is no support for vrfanservice.
https://www.sexlikereal.com/studios/vr-fan-service

**Comment by otheraccount3** [comment]
can you add VRpornjack on SLR?

https://www.sexlikereal.com/studios/vrpornjack

**Comment by Deflated-Criossant** [comment]
New site
https://virtualporn.com/

eg: link to scene
https://virtualporn.com/video3410699/blake-blossom-blooms-in-vr

**Comment by theRealKLH** [comment]
> New site
> https://virtualporn.com/
> 
> eg: link to scene
> https://virtualporn.com/video3410699/blake-blossom-blooms-in-vr

Do you have the filename formats for the different resolutions?

**Comment by theRealKLH** [comment]
> BigTitsinVR and http://realteensvr.com/ are now part of NA and sites redirect to NaughtyAmerica

and those scenes are "gone forever".. at least for now, there's no way to get to "existing scenes" on NA (at least for RealTeens).

**Comment by theRealKLH** [comment]
> can you add VRpornjack on SLR?
> 
> https://www.sexlikereal.com/studios/vrpornjack

Added in #562

**Comment by theRealKLH** [comment]
@D

> > New site
> > https://virtualporn.com/
> > eg: link to scene
> > https://virtualporn.com/video3410699/blake-blossom-blooms-in-vr
> 
> Do you have the filename formats for the different resolutions?

@Deflated-Criossant Not sure if you saw this ^^^^^

**Comment by vt-idiot** [comment]
1. "Manual entry" scraper please, give me a box to fill out or even a JSON template to work with. #442 
2. There are some JAV studios on SLR that would be helpful for filling in the gaps, like KoalaVR, KMPVR, MAX-A, and PRESTIGE, amongst others. But particularly PRESTIGE.
3. Afesta?
4. Additional/optional SLR "fallbacks"

`https://www.sexlikereal.com/studios/kmpvr`
`https://www.sexlikereal.com/studios/koalavr`
`https://www.sexlikereal.com/studios/max-a`
`https://www.sexlikereal.com/studios/prestige`

TMAVR was very helpful to have included as there are a few videos that aren't available on R18 but do show up on SLR. It's odd that SOD Create is included when most if not all of their content overlaps with R18 - meaning there's still "missing" 3DSVR/SOD content that's "Japan only" when in reality it is available on Afesta

<details><summary>I don't know if it'd be possible to add Afesta as a scraper, but I hope this helps.</summary>

This `https://www.afesta.tv/en_vr/face.php?fid=93f4694e1fa104a7982c9542eecf55b8` is a link to 3DVSR-1021, which you'd only realize by looking at the thumbnail URL once the page loaded: `https://www.afesta.tv/upimg/imgsod12345/3DSVR-1021-R1.jpg` It doesn't show up on R18, and searching for either actress on SLR, Mio Ichijou or Noa Eikawa, turns up nothing. It's only on Afesta.

The movie filenames are usually as follows, `3DVSR-1021_R1.mp4` `3DVSR-1021_R2.mp4` etc. or maybe it was `-R1.mp4` - I forget. Raw search requests sent to `https://www.afesta.tv/en_vr/search.php` look like this: `co_folder_cid%5B0%5D=&keyword=3DSVR-1021&header=search`

Of course, being able to *manually* scrape a database like javbus for whatever R18 is missing would be easiest and likely quickest, the URLs are always very straightforward: `https://www.javbus.com/en/3DSVR-1021`

The ~100 most recent 3DSVR videos are otherwise unavailable, as are random videos from KMPVR (several different DVD prefixes) and PRESTIGE (also several). There are some PRESTIGE videos that have "conflicting" IDs, like PRVR-026 with Ryou Harusaki:
* Afesta `https://www.afesta.tv/en_vr/face.php?fid=26aabfeefb29201bb1e17daf6f703922`
* SLR `https://www.sexlikereal.com/scenes/ryou-harusaki-dont-make-a-sound-whispering-nurse-and-forbidden-sex-20620`

javbus and R18 show a different video with Kira Rin from a different studio:

* R18 `https://www.r18.com/videos/vod/movies/detail/-/id=prvr00026/`
* javbus `https://www.javbus.com/en/PRVR-026`

because the correct DVD ID for the PRESTIGE scene would be PR*D*VR-026 - there are no PRESTIGE AV on R18 at all AFAIK
* javbus (correct) `https://www.javbus.com/en/PRDVR-026`
</details>

There are lots of things that ought to be scraped from SLR - e.g. the VRBangers scraper hasn't been working at all on my end and I can't seem to figure out if I've done something wrong or been blocked/throttled.

**Comment by claforest** [comment]
it looks like StripzVR also has their own site outside of SLR, could we get that added?
https://www.stripzvr.com/

**Comment by legitswitch** [comment]
Newer 3DSVR videos (sod create) are now exclusive to mgstage.com

Please add scraper support as you cannot find the newer scenes on R18 anymore

**Comment by theRealKLH** [comment]
> it looks like StripzVR also has their own site outside of SLR, could we get that added? https://www.stripzvr.com/

StripzVR has always had it's own site. I believe the reason it is not used is due to lack of data such as tags.

**Comment by vt-idiot** [comment]
> Newer 3DSVR videos (sod create) are now exclusive to mgstage.com
> 
> Please add scraper support as you cannot find the newer scenes on R18 anymore
Did they stop posting them on Afesta?

**Comment by legitswitch** [comment]
Never used it, but I just checked and they are still posted there. So Afesta and MGSTAGE both have the newer vids.

Looks like it's both DMM and R18 that don't have them anymore.

**Comment by otheraccount3** [comment]
Any chance we can get scrapers for fuckpassvr and vrixxen on SLR?

**Comment by str0keR** [comment]
+1 for stripzvr.com - SLR don't have a lot of their catalog.

**Comment by ew12ght** [comment]
Can you add [www.pirouvr.com](www.pirouvr.com) ?

**Comment by ew12ght** [comment]
New site

https://virtualporn.com/

**Comment by MrChristopher360** [comment]
Looks like Brazzers has finally entered the ring!

**Comment by theRealKLH** [comment]
> Looks like Brazzers has finally entered the ring!

I heard that the Brazzers scene (Abigail Mac) that was released was "Ass"!

**Comment by zirenzhou** [comment]
JavDB is an Online Jav Database offer much more records than r18.com
If XBVR Could add it to Scrapers would be great
https://javdb36.com/

**Comment by Runisco** [comment]
https://deviantsvr.com/

**Comment by s0829964** [comment]
As R18 is closing down. 
JAVLibrary might be a way out. 
Here is the link for JVR 
[https://www.javlibrary.com/tw/vl_genre.php?g=aaua](https://www.javlibrary.com/tw/vl_genre.php?g=aaua)

**Comment by LucasianProfessor** [comment]
https://www.realpornstarsvr.com/

**Comment by gnadenlos** [comment]
SLR_SinsVR (as the collection is very different from the official site)
FuckPassVR (studio site - SLR is missing multiple scenes and is not up-to-date)

**Comment by Juv-ax** [comment]
Request:

https://jav.land for scraping Jav VR. 

This site uses the same tags as r18 and dmm and is updated before any others (like javdatabase, javdatabase, javbus ect).

It also allows searching by movie id or content id and has the same complete preview images as dmm.

https://jav.land/en/id_search.php?keys=tmavr-180
https://jav.land/en/id_search.php?keys=55tmavr00180

**Comment by thebrnd** [comment]
@s0829964 javlibrary is added in pull request #1100
@Juv-ax jav.land is added in pull request #1100 now as well, ready to be tested. It looks like it has some tags that are cut off or difficult to understand like "m man" but if that means something to other people then great. The actress names appear to be different on every single site as well.

**Comment by vt-idiot** [comment]
> It looks like it has some tags that are cut off or difficult to understand like "m man"

R18's old `masochist man` tag, probably only gets used with shorthand "M" in Japanese

**Comment by RKmaster1324** [comment]
Here is one I found.

Dreamcam from SLR is not in scrapper.

**Comment by Lunco** [comment]
is WetVR scraper broken atm? Can't get any scenes to work.

**Comment by theRealKLH** [comment]
> is WetVR scraper broken atm? Can't get any scenes to work. 

What exactly is not working?

**Comment by blamezippy** [comment]
> > is WetVR scraper broken atm? Can't get any scenes to work.
> 
> What exactly is not working?

The scraper runs, find scenes, but does not add them.

**Comment by Xplode85** [comment]
A few studios on SLR missing:

https://www.sexlikereal.com/studios/vrplayful
https://www.sexlikereal.com/studios/kinkykingdom
https://www.sexlikereal.com/studios/vrhard
https://www.sexlikereal.com/studios/all-anal-vr
https://www.sexlikereal.com/studios/vrmansion
https://www.sexlikereal.com/studios/putalocuravr
https://www.sexlikereal.com/studios/virtual-papi
https://www.sexlikereal.com/studios/yogavr

**Comment by theRealKLH** [comment]
You need to update the `custom.json` with any missing studios on SLR. The scraper is now only updated to fix functionality. Same with studios listed on POVR.

**Comment by Xplode85** [comment]
> You need to update the `custom.json` with any missing studios on SLR. The scraper is now only updated to fix functionality. Same with studios listed on POVR.

Do you have any more info regarding this? I can't find any documentation on custom.json or a file with that name.

**Comment by Xplode85** [comment]
> > You need to update the `custom.json` with any missing studios on SLR. The scraper is now only updated to fix functionality. Same with studios listed on POVR.
> 
> Do you have any more info regarding this? I can't find any documentation on custom.json or a file with that name.

Ok i found how to do this myself, it's not a file to edit manually, but has to be added from the UI under Options -> Interfaces -> Advanced -> Create Custom Site

**Comment by theRealKLH** [comment]
You can most definitely update the custom.json. Please do a search here for instruction. Currently on my phone or I would give you specific link to the info.

**Comment by guillorec** [comment]
yogavr
blowjobnow

**Comment by Sunset-1982** [comment]
It would be nice if Steel R and PutalocuraVR from SLR would be added...

Is there more information about the custom.json update anywhere? I couldn't find anything.

**Comment by Sunset-1982** [comment]
> You can most definitely update the custom.json. Please do a search here for instruction. Currently on my phone or I would give you specific link to the info.

Couldn't find any more info about it...

**Comment by moToroTor** [comment]
OnlyTarts seems to not be on aggregators or I would use those to scrape. Seems logical to just ask here instead of opening a new ticket.
